AzureCalling/External/Calling/AudioDeviceSelectionManager.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+import AVFoundation
+
+class AudioDeviceSelectionManager {
+
+    // MARK: Public Functions
+
+    public func initAudioDevicess() -> [AudioDeviceDataModel] {
```

Comment:
At the moment, our audio override options are either `.none` or `.speaker`.  When set to `.none`, this covers the iPhone earpiece, or any bluetooth speaker/headset, with the bluetooth headset taking priority over the iPhone earpiece.  I see that it might be possible to more specifically select these outputs as they become available, but I would suggest that they be a separate ticket.